### ISSUE TITLE
longer prompt 

### ISSUE THREAD

**Issue by yanmengxiang123456899** [h2oai/h2ogpt#744]

**Comment by yanmengxiang123456899** [comment]
I don't want the precision of the model to be int-8 or int-4.Is there any way to solve it for longer prompt？Thanks！

**Comment by pseudotensor** [comment]
Yes, linear factor 4 will use about 4x as much memory, so probably not possible for 70B with each board of 23GB.

@arnocandel thoughts?

**Comment by yanmengxiang123456899** [comment]
Like #692.
If we can do distributed generation with > 8 GPUs, maybe it can solve.
